src/main/java/org/zalando/pazuzu/feature/Tag/Tag.java

```diff
@@ -0,0 +1,60 @@
+package org.zalando.pazuzu.feature.tag;
+
+import javax.persistence.*;
+
+/**
+ * Created by vpavlyshyn on 09/06/16.
+ */
+@Entity
+public class Tag {
+    public Tag(String name) {
```

Comment:
Common code style in this repo is: line break after class definition. Please stick to it.